smithy-aws-traits/src/main/java/software/amazon/smithy/aws/traits/protocols/AwsQueryCompatibleTrait.java

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright 2021 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Comment:
```suggestion
 * Copyright 2022 Amazon.com, Inc. or its affiliates. All Rights Reserved.
```

Reply:
Can you please update the year as Michael suggested?